Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FarePolicy/FarePolicyProgressiveDetails.hs

```diff
@@ -33,32 +36,14 @@ data FPProgressiveDetailsD (s :: UsageSafety) = FPProgressiveDetails
     baseDistance :: Meters,
     distanceUnit :: DistanceUnit,
     perExtraKmRateSections :: NonEmpty (FPProgressiveDetailsPerExtraKmRateSectionD s),
+    perMinRateSections :: [FPProgressiveDetailsPerMinRateSection],
```

Comment:
It is confusing, as one Progressive Fare policy couldn't have 2 sections, one for per minute rate and other for per km rate. Either it would be per minute or per km.

I would suggest create one more table and call it `FPProgressiveTimeBasedDetails`. 
Because anyways, I don't think for per minute fare policy we will be have baseDistance and deadKmFare. It would be most probably baseTime etc. This will also provides us more flexibility to define more params specific to time based fare policy instead of mixing both time and distance based things.